docs/user-guides/collect-systemd-logs.md

```diff
@@ -0,0 +1,34 @@
+# Collect Service Logs of Kubernetes Nodes
+
+This tutorial guides you on how to collect Service logs of kubernetes nodes with fluentbit operator.
+
+# Demo
+
+First, install or update the fluentbit operator
+
+```shell
+kubectl apply -f manifests/setup/
+kubectl apply -f manifests/logging-stack/fluentbit-fluentBit.yaml
+kubectl apply -f manifests/logging-stack/fluentbitconfig-fluentBitConfig.yaml
+```
+
+Second, change the service logs directory
+
+```shell
+mkdir /var/log/journal/
```

Comment:
Why do we need to recreate the journal directory?

Reply:
The default directory is /run/log/, if use the default directory, the fluent bit can not keep track of Journald cursor.

Reply:
in our dev env, there is already an existing `/var/log/journal` , so maybe we can describe this like `Please create directory /var/log/journal if it doesn't exist:`
![image](https://user-images.githubusercontent.com/18525465/103071171-069e9d00-45fe-11eb-9e72-4a7ff16da93b.png)